skills/csharp/calendarskill/Models/EventModel.cs

```diff
@@ -392,15 +394,15 @@ public DateTime StartTime
                             msftEventData.Start = new Microsoft.Graph.DateTimeTimeZone();
                         }
 
-                        msftEventData.Start.DateTime = value.ToString("o");
+                        msftEventData.Start.DateTime = value.ToString("yyyy-MM-ddTHH:mm:ss");
```

Comment:
>yyyy-MM-ddTHH:mm:ss [](start = 71, length = 19)

What's the purpose behind changing the format here? Generally speaking, it is sensible to stick to ISO-8601 formatting for date time strings.

Reply:
The value is a UTC time. After tostring("o"), it will be convert to a string with format like "yyyy-MM-ddTHH:mm:ssZ". The msgraph api will treat the time as an UTC time and ignore the timezone we set.